src/main.rs

```diff
@@ -67,9 +68,9 @@ fn setup_logging(verbose: bool) {
 }
 
 fn list_backends() {
-    println!("Available backends : ");
+    println!("Available backends: ");
     for (&(name, _), idx) in BACKENDS.iter().zip(0..) {
-        if idx == 0 {
+        if matches!(idx, 0) {
```

Comment:
Is this something that clippy wanted? You're not matching against an enum are you?

Reply:
I just went thought and changed all == to matches that I could without major changes I suppose == still makes logical sense for numbers but for non numbers I think I prefer matches. I can change back to == for number type equality if you like.

Reply:
I changed all number comparisons back to ==.